server/apps/postgres-app/src/main/java/org/apache/james/PostgresJamesConfiguration.java

```diff
@@ -42,9 +42,9 @@
 
 public class PostgresJamesConfiguration implements Configuration {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(PostgresJamesConfiguration.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger("org.apache.james.CONFIGURATION");
```

Comment:
Q: why change this?

Reply:
the LOGGER should be `org.apache.james.CONFIGURATION`, it will be easier groupBy when init start server